server_development/topics/admin-rest-api.adoc

```diff
@@ -10,7 +10,32 @@ A token can be obtained by enabling authenticating to your application with {pro
 
 For complete documentation see {apidocs_link}[{apidocs_name}].
 
-=== Example using CURL
+=== Examples using CURL
+
+==== Authenticating with `client_id` and `client_secret`
```

Comment:
I'd change this to "Authenticate with a service account" and move it below the "Authenticate with username and password" example